tests/snapshots.rs

```diff
@@ -0,0 +1,124 @@
+use serde::Deserialize;
```

Comment:
so this file is based on `convert.rs`, right?
Do we still need `convert.rs` then? It seems to me that the snapshot tests could ultimately replace it.

Reply:
Yeah I don't think we'll need `convert.rs` once we're using snapshots instead. We could remove it in this PR or wait until we have snapshots for some of the other paths (spv-in, glsl-in)

Reply:
`tests/convert.rs` has been removed now